### DISCUSSION TITLE
SmarterASP.NET and Oqtane 5.1.1

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#4151] [General]
I just upgraded [https://LearnOqtane.com](https://LearnOqtane.com) to Oqtane 5.1.1 from 5.1 which is hosted on [SmarterASP.NET](https://www.SmarterASP.NET/index?r=vnetonline)

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/43329595-d970-444e-98cd-ab39c06b052a)

The built-in update method didn't work for me, so I did the upgrade manually. 

These are the Steps I followed:

1. Took a backup of DB
2. Took a backup of all the files in my website folder 
3. Turned of the site in my website Control Panel

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/173b305f-271c-4d08-9feb-94f740c20980)

4. Clicked on Advanced -> Pool Manager -> Actions and changed Pool to 64 bit

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/79b4417e-7669-4777-970d-4105e7926fdf)

5. Turned the site back on, navigated to [https://LearnOqtane.com](https://LearnOqtane.com)

**Comment by leigh-pointer** [comment]
I had no problems upgrading all my instances using the built in framework upgrade option.

**Comment by vnetonline** [comment]
@leigh-pointer Can you explain the process you went through? What plan are you on? What is your application pool size?

Are you hosting your website with [SmarterASP.NET](https://www.smarterasp.net/index?r=vnetonline)

**Reply by leigh-pointer** [reply]
I'm using Interserver, MonsterASP and local IIS.  
Process:
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/c305d0be-20dd-487c-9071-1f544637b934)
Click Download 5.1.1

![image](https://github.com/oqtane/oqtane.framework/assets/9447612/8c1b6f71-477d-4922-a6a4-01fd81f83a8c)
Click Upgrade and wait .......................................................................................................
Can take a while, and when you think has hung, believe me just wait some more.

That is it!
